### PR TITLE
[TRUNK-5274] Delete unused imports

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/TRUNK-5274

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateUserDAO.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ public boolean hasDuplicateUsername(String username, String systemId, Integer us
 		}
 		
 		if (userId == null) {
-			userId = Integer.valueOf(-1);
+			userId = -1;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateUserDAO.java]: Is this also about unused imports?

File: api/src/main/java/org/openmrs/util/FormUtil.java [hunk 2]
```diff
@@ -159,7 +159,7 @@ public static String getNewTag(String s, ArrayList<String> tagList) {
 	 */
 	public static Map<Integer, TreeSet<FormField>> getFormStructure(Form form) {
 		Map<Integer, TreeSet<FormField>> formStructure = new TreeMap<>();
-		Integer base = Integer.valueOf(0);
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/util/FormUtil.java]: Same here and many other places in this commit.

**Comment by BartlomiejRasztabiga** [comment]
I see, I had to merge two commits. I will fix that

**Comment by dkayiwa** [comment]
@BartlomiejRasztabiga are you still working on this?

**Comment by BartlomiejRasztabiga** [comment]
Yes, done :)

**Pull request merged**